components/homme/src/share/cxx/utilities/scream_tridiag.hpp

```diff
@@ -2,7 +2,7 @@
 #define INCLUDE_SCREAM_TRIDIAG
 
 /* Warning: This file is a copy of
-      components/scream/src/share/util/scream_tridiag.hpp
+      components/eamxx/src/share/util/scream_tridiag.hpp
```

Comment:
It would be best to leave this file out since it's in components/homme, which we strongly prefer to modify upstream.

Reply:
Good call. Reverted.